tests/TenantDatabaseManagerTest.php

```diff
@@ -390,6 +390,81 @@
     expect(file_exists($customPath . '/' . $name))->toBeTrue();
 });
 
+test('template tenant connection value can be connection name or connection array', function () {
```

Comment:
I think the name should suggest that the `connection` can be either the name of an existing configured connection template, and that instead of using that existing template, you can also define a new one

```suggestion
test('you can define the connection template array in template_tenant_connection or use a name of an existing connection template', function () {
```

Something like this. Do you agree?